src/CodeGeneration.Roslyn.Tasks/build/net20/CodeGeneration.Roslyn.BuildTime.props

```diff
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="utf-8" ?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\$(MSBuildThisFileName)$(MSBuildThisFileExtension)" />
```

Comment:
Have you confirmed that with the removal of all these files that the resulting package still correctly installs into projects with all these target frameworks?
I added them because (at least at the time) nuget would consider a top-level build file as a net40 targeted file, so it wouldn't add the import to net20 or other target framework projects.

Reply:
I've confirmed that (after my incoming fixes) it works for net20, net40, net45, netstandard1.0 and netstandard 2.0. Portable targets aren't supported, but it can be probably done, if necessary.

Reply:
It'll probably require using MsBuildSdkExtras package for multi-targeting portables